Common/GPU/Vulkan/VulkanRenderManager.cpp

```diff
@@ -972,7 +975,8 @@ bool VulkanRenderManager::CopyFramebufferToMemory(VKRFramebuffer *src, VkImageAs
 	}
 
 	// Need to call this after FlushSync so the pixels are guaranteed to be ready in CPU-accessible VRAM.
-	queueRunner_.CopyReadbackBuffer(src, w, h, srcFormat, destFormat, pixelStride, pixels);
+	queueRunner_.CopyReadbackBuffer(frameData_[vulkan_->GetCurFrame()],
+		mode == Draw::ReadbackMode::OLD_DATA_OK ? src : nullptr, w, h, srcFormat, destFormat, pixelStride, pixels);
 	return true;
```

Comment:
Shouldn't this probably care about the `CopyReadbackBuffer` return value?

-[Unknown]

Reply:
Yeah, I'll fix. Though in many cases, once you step back high enough into the callstack, nobody cares about the return value anyway and it's a little unclear what can be done, but of course we should do it right here.